File: chapter-03/exercise-09-paint-calculator/PaintCalculator.java
class PaintCalculator {
  private static final int ONE_GALLON = 350;
  
  public static void main(String[] args) {
    var input = System.console().readLine("What is length of the ceiling? ");
    var length = Integer.parseInt(input);

    input = System.console().readLine("What is the width of the ceiling? ");
    var width = Integer.parseInt(input);

    var area = length * width;
    var gallonsNeeded = area / ONE_GALLON;
    if (area % ONE_GALLON > 0) {
      gallonsNeeded++;
    }

    System.out.printf("%nYou will need to purchase %d gallons of paint to cover %d square feet.%n", gallonsNeeded, area);
  }
}

File: 09-paint-calculator/paint_calculator.py
ONE_GALLON = 350

length = int(input("What is the length of the ceiling? "))
width = int(input("What is the width of the ceiling? "))

area = length * width
gallons = area // ONE_GALLON
if area % ONE_GALLON > 0:
    gallons += 1

print(f"You will need to purchase {gallons} gallons of\n"
      f"paint to cover {area} square feet.")

